Rock.Messaging/MQ/NamedPipes/NamedPipeConfig.cs

```diff
@@ -1,4 +1,6 @@
 ﻿#if ROCKLIB
+using System;
```

Comment:
I think this will be an issue in non-rocklib.  The added ArgumentNullException will show up in non-rocklib, but the using system won't.

Reply:
Good catch.